src/components/ListItems/UserListItem/index.tsx

```
@@ -46,7 +46,8 @@ export const UserListItem = memo(
       <View style={styles.container} {...stopPropagation}>
         <Touchable
           style={styles.touchArea}
-          onPress={() => navigation.navigate('UserProfile', {userId})}>
+          // TODO: temp profile disabling
+          onPress={() => false && navigation.navigate('UserProfile', {userId})}>
```

I may suggest just to comment UserProfile from Navigator. All the places will silently fail to navigate

---

Why is it better? We'll get errors in sentry...

---

Oh, I see. Then OK